### ISSUE TITLE
key fails to issue new codes after resume from sleep

### ISSUE THREAD

**Issue by bcbv** [Yubico/yubioath-desktop#95]
If Yubikey is inserted, and Yubico Authenticator is running and when the laptop goes to sleep, when resuming from sleep, the Yubico app does not continue to get new OTP codes. (Can also see this because the timer bar at the top of the app is frozen.)

It would be nice if the app would attempt to reconnect to the Yubikey if the timer bar is stopped or no OTP codes are generated within a minute or two.

The OS for this bug is 10.11.6.
Yubikey application version: 3.1.24
Library version: 1.17.3
Firmware version: 4.2.8

Yubikey authenticator version: 2.3.0
ykpers: 1.17.2

**Comment by ralphm** [comment]
I'm experiencing the same thing on Ubuntu 16.04.

**Comment by msusta** [comment]
I'm also experiencing this. Also when this happens it doesn't help me if I remove the token from USB and insert it - I have to reboot to get things working which is pretty depressing.

OS: Windows 7 SP 1 (enterprise)
Firmware: 4.3.1
Yubico Authenticator: 3.0.1
ykpers: 1.17.3

**Comment by Fuzzillogic** [comment]
Here too, but not only after resuming from standby. It really is a hit-or-miss every time on this machine. Keeping the authenticator running, or restarting it, sometimes yields a new OTP after a while. Other functionalities work just fine; I've just logged in using U2F.

OS: Windows 7 SP 1 (enterprise)
Yubico Authenticator: 3.0.1
ykpers: 1.17.3

I've installed latest Intel drivers, latest BIOS update (A20 for this Dell Precision T1700, prior updates improved on USB and resuming, but made no difference)

My home desktop, Intel P67 chipset, running Ubuntu 16.10, has no problems at all with this Yubikey. The authenticator (also 3.0.1, from the Yubico PPA) just works.
